changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "1.1.2-beta5": [
+      "[Added] Syntax highlighting for java server pages (jsp) - #4470. Thanks @damaneice!",
+      "[Fixed] Ahead/behind info is not remembered when selecting branch to compare  - #4517",
```

Comment:
i'd be open to making this an "Improved" since it wasn't necessarily wrong but we're making an optimization. thoughts?

Reply:
Yea, I can get behind marking that as an improvement.